packages/ComponentLibrary/src/components/Nav/Nav.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import SearchIcon from '@mui/icons-material/Search';
+import { NavStyles, boxStyles } from './Nav.styles';
+import RigthButtons from './RigthButtons/Nav.RightButtons';
+import Logo from './LeftComponents/Logo';
+import { AppIcon } from './LeftComponents/App.Icon';
+
+const Nav: React.FC = () => {
+  return (
+    <div style={boxStyles}>
+      <AppIcon />
+      <nav style={NavStyles}>
+        <Logo />
+        <SearchIcon />
+        <div>
```

Comment:
Is there a reason why you need this div?

Shouldn't it be inside the component and as a React Fragment?